Front-End/src/PageM.js

```diff
@@ -52,585 +53,571 @@ function SignIn(props) {
-                        </tr>
-                        <tr>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                        </tr>
-                        <tr>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                        </tr>
-                        <tr>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                        </tr>
-                        <tr>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                            <td>Hello</td>
-                        </tr>
-                    </table>
+                                    <th> Thickness (Extrusion)</th>
+                                    <th> Thickness (Plate)</th>
+                                    <th> Material (Plate)</th>
+                                    <th>  Length in Inches</th>
+                                    <th>  Part Number</th>
+                                    <th> Date</th>
+                                </tr>
+                                <tr>
+                                    <td>{advice[0][0]}</td>
+                                    <td>{advice[0][1]}</td>
+                                    <td>{advice[0][2]}</td>
+                                    <td>{advice[0][3]}</td>
+                                    <td>{advice[0][4]}</td>
+                                    <td>{advice[0][5]}</td>
+                                    <td>{advice[0][6]}</td>
+                                    <td>{advice[0][7]}</td>
+                                    <td>{advice[0][8]}</td>
+                                    <td>{advice[0][9]}</td>
+                                </tr>
```

Comment:
Since we're using `advice.map()` below this, we won't need to manually add data from our rows like this 😃 

Let's remove the `<tr>(...)</tr>` from lines 177 to 188 (just where we have `{advice[0][*]}`)

Reply:
fixed